dm/relay/relay_test.go

```diff
@@ -453,14 +455,44 @@ func (t *testRelaySuite) TestHandleEvent(c *C) {
 		c.Assert(errors.Cause(handleErr), Equals, reader2.err)
 	}
 
+	// reader return fake rotate event
+	reader2.err = nil
+	reader2.result.Event = fakeRotateEv
+	// writer return error to force handleEvents return
+	writer2.err = errors.New("writer error for testing")
+	// return with the annotated writer error
+	_, err = r.handleEvents(ctx, reader2, transformer2, writer2)
+	c.Assert(errors.Cause(err), Equals, writer2.err)
+	// after handle rotate event, we save and flush the meta immediately
+	c.Assert(r.meta.Dirty(), Equals, false)
+	{
+		lm := r.meta.(*LocalMeta)
+		c.Assert(lm.BinLogName, Equals, "mysql-bin.666888")
```

Comment:
```suggestion
		c.Assert(lm.BinLogName, Equals, binlogPos.Name)
```
ditto in L476
and i can not understand why `lm.BinLogPos, Equals, uint32(1234)` , what is `uint32(1234)`?

Reply:
see line 427